components/commentCard.jsx

```diff
@@ -0,0 +1,10 @@
+export function CommentCard({ comment }) {
+  return (
+    <>
```

Comment:
As you're returning the singular li element, wrapping that in a fragment is giving unnecessary nesting.